src/components/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useContext } from 'react';
+import {
+  Button,
+  Box,
+  FormControl,
+  Input,
+  FormLabel,
+  Text,
+  VStack,
+} from '@chakra-ui/react';
+import { FirebaseContext } from '../../services/firebase/firebase';
+
+const SignIn = () => {
```

Comment:
Since we have a SignIn page as well as a SignIn Form component, we should just call this SignInForm, and similarly for SignUpForm

Reply:
OK